### ISSUE TITLE
Help with carousel in tabbed page

### ISSUE THREAD

**Issue by ThothsCoderDev** [AndreiMisiukevich/CardView#47]
I am using your carousel in a page within a tabbed page but when i try to swipe the carousel it changes tabs I may be doing something wrong but it would mean a lot if you could help me.
Thanks

**Comment by AndreiMisiukevich** [comment]
Hi) thank you for feedback
I think, you should switch off tabbed page swipe animation, because app can't understand, which animation it should strat.

Could you try this work around? You will have to create custom renderer and disable swipes there

**Comment by thedjnova** [comment]
Agreed, tabbedpage and cardview cant exist together without disabling the swipe on one or the other. Nothing we can really do aboit it either, they share the same gesture.

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
